### ISSUE TITLE
Some BLFont APIs belong to BLFontFace

### ISSUE THREAD

**Issue by neslib** [blend2d/blend2d#69]
There are some `BLFont` API's that don't apply to a specific font, but to the face used by the font.

For example, the `BLFont::mapTextToGlyphs` API is independent of the font, but only depends on the face that is used by the font.

It's fine to have this API at the `BLFont` level, but it would be nice to have a `BLFontFace::mapTextToGlyphs` API as well, so we can use it without having to create a `BLFont` instance.

**Comment by kobalicek** [comment]
This is right. I will definitely think of this in upcoming weeks as text is now a priority feature.
